irohad/main/server_runner.hpp

```diff
@@ -26,13 +27,38 @@ namespace torii {
   class ToriiServiceHandler;
 }
 
+/**
+ * Class runs Torii server for handling queries and commands.
+ */
 class ServerRunner {
  public:
+  /**
+   * Constructor. Initialize a new instance of ServerRunner class.
+   * @param address - the address the server will be bind to in URI form.
+   */
   explicit ServerRunner(const std::string &address);
+
+  /**
+   * Destructor. Shutdowns server.
+   */
   ~ServerRunner();
+
+  /**
+   * Initialize server and runs main loop.
```

Comment:
Better to be consistent in form of verbs, use simple form without 's', like :  Initialise server and run main loop